libres/lib/enkf/enkf_obs.cpp

```diff
@@ -659,18 +659,14 @@ static void handle_general_observation(enkf_obs_type *enkf_obs,
     stringlist_free(block_obs_keys);
 }
 
-static void enkf_obs_reinterpret_DT_FILE(const char *config_file) {
+static void enkf_obs_reinterpret_DT_FILE(const char *errors) {
     // clang-format off
-    fprintf(stderr, "**********************************************************************\n");
-    fprintf(stderr, "* In ert version 2.3 we have changed how filepaths are interpreted   *\n");
-    fprintf(stderr, "* in the observation file. When using the keywords OBS_FILE,         *\n");
-    fprintf(stderr, "* ERROR_COVAR and INDEX_FILE in the category GENERAL_OBSERVATION     *\n");
-    fprintf(stderr, "* the filenames will be interpreted relative to the main observation *\n");
-    fprintf(stderr, "* file.                                                              *\n");
-    fprintf(stderr, "*                                                                    *\n");
-    fprintf(stderr, "* Please update the OBS_FILE, ERROR_COVAR and INDEX_FILE keywords    *\n");
-    fprintf(stderr, "* by removing the path to the main observation file.                 *\n");
-    fprintf(stderr, "**********************************************************************\n");
+    fprintf(stderr, "*****************************************\n");
```

Comment:
Increase number of stars to cover whole text? ⭐

Reply:
A bit hard because possible varying length of resolved filenames (which is why I removed that nice feature in the first place) 😄  .

(It is possible to do, of-course: Split the string on newline + loop over result to find max-length. IMO not worth the trouble, but can easily be stitched together if required.)

Reply:
I agree that it's not worth the trouble 👍